src/index.ts

```diff
@@ -179,6 +198,9 @@ export async function createClient(userConfig: UserConfig): Promise<Client> {
     }
 
     console.log('✨ Done! Your client is located in:', config.output);
+
+    logMissingDependenciesWarning(config.client, dependencies);
```

Comment:
Can this be moved after the first client log call? Since it's related to that setting